models/user.js

```diff
@@ -0,0 +1 @@
+//users model
```

Comment:
are you think user is best name for student model?

Reply:
yes user can login/register and make some blogs and buy courses
so user is best name for student collection✋